src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,97 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int INITIAL_CAPACITY = 10;
+    private static final double INCREMENT_CAPACITY = 1.5;
+    private int size;
+    private T[] elements;
+    private int arrayCapacity;
```

Comment:
we don't need this variable. We have `size`, which represents a number of elements in our List, and we cal call `elements.length` if we want to know length of array

Reply:
But elements.length () gives us all the length, including the empty field in the array.